### PR TITLE
LPS-196385 Show permission message in the permission section of DLFileEntry based on Parent folder propagation setting

### PR DESCRIPTION
**Pull request by ces-locpham**
Hi @tototrinh ,
Please help me review this ticket,
Thanks,
Loc

### CODE REVIEW

File: modules/apps/document-library/document-library-video/src/main/java/com/liferay/document/library/video/internal/display/context/DLVideoExternalShortcutDLEditFileEntryDisplayContext.java [hunk 1]
```diff
@@ -62,6 +62,24 @@ public DLFilePicker getDLFilePicker(String onFilePickCallback) {
...
+
+	@Override
+	public String getInheritableParentFolderName() {
+		if (parentDisplayContext == null) {
+			return null;
+		}
+
+		return parentDisplayContext.getInheritableParentFolderName();
+	}
+
```
Comment (tototrinh) [inline modules/apps/document-library/document-library-video/src/main/java/com/liferay/document/library/video/internal/display/context/DLVideoExternalShortcutDLEditFileEntryDisplayContext.java:82]: I think these codes should be added in BaseDLEditFileEntryDisplayContext instead. And looks like we don't need to check null here. Thanks.

**Review by ces-totrinh — APPROVED** [review]
LGTM. Thanks

**Review by ces-luannguyen — APPROVED** [review]

**Review by ces-quanhuynh — APPROVED** [review]

**Comment by ces-totrinh** [comment]
@hhtank, this PR passed our internal review. Please help take a look and give your feedback. Thanks.

**Comment by hhtank** [comment]
@tototrinh LGTM, thanks.

**Comment by ces-totrinh** [comment]
Merged. Thanks all!

**Pull request merged**